languagetool-language-modules/de/src/test/java/org/languagetool/rules/de/AgreementRuleTest.java

```diff
@@ -105,6 +105,11 @@ public void testDetNounRule() throws IOException {
     assertGood("Das Dach von meinen Autos.");
     assertGood("Da stellt sich die Frage: Ist das Science-Fiction oder moderne Mobilität?");
 
+    assertGood("Wir machen das Januar.");
+    assertGood("Wir teilen das Morgen mit.");
+    assertGood("Wir präsentierten das vorletzten Sonnabend.");
+    assertGood("Wir releasen das Vormittags.");
```

Comment:
"Morgen" und "Vormittags" werden in dem Kontext wahrscheinlich klein geschrieben. Daher brauchen wir hier irgendwann mal eine Regel, die das erkennt.